messageapp/src/clients/updateMessage.js

```diff
@@ -0,0 +1,36 @@
+import { Message, MessageBackup } from "../models/message.js";
+
+import updateBudget from "./updateBudget.js";
+
+import lockedSync from "locked-sync"
+const sync = lockedSync()
```

Comment:
Think about the library that you are using `locked-sync`. Is it right implemented?

Look at the docs:

https://www.npmjs.com/package/locked-sync